rebasehelper/build_helper.py

```diff
@@ -283,7 +283,7 @@ class MockBuildTool(BuildToolBase):
     logs = []
 
     @classmethod
-    def _build_rpm(cls, srpm, results_dir, root=None, arch=None):
+    def _build_rpm(cls, srpm, results_dir, root=None, arch=None, enable_option=None, **kwargs):
```

Comment:
You won't because the option has default value. Getting error which we can fix is better than masking it. For new code explicit parameters are preferred, unless there is strong reason to take arbitrary kwargs. If there will be such situation in the future, we will add it then, now it is just dead code.